### PR TITLE
[FEAT] Unified remote APIs - Prompt studio changes

### PR DESCRIPTION
**Pull request by harini-venkataraman**
## What

-

## Why

-

## How

-

## Can this PR break any existing features. If yes, please list possible items. If no, please explain why. (PS: Admins do not merge the PR without this section filled)

-

## Database Migrations

- 

## Env Config

- 

## Relevant Docs

-

## Related Issues or PRs

-

## Dependencies Versions

-

## Notes on Testing

-

## Checklist

I have read and understood the [Contribution Guidelines]().

### CODE REVIEW

File: backend/utils/file_storage_helper.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    def initialize_file_storage(type: FileStorageType) -> FileStorage:
+        provider_data = FileStorageUtil.load_file_storage_envs()
+        provider = provider_data[FileStorageKeys.FILE_STORAGE_PROVIDER]
+        credentials = provider_data[FileStorageKeys.FILE_STORAGE_CREDENTIALS]
+        if type.value == FileStorageType.PERMANENT.value:
+            file_storage = PermanentFileStorage(
+                provider=provider, credentials=credentials
+            )
+        if type.value == FileStorageType.DEFAULT.value:
+            file_storage = FileStorage(provider=provider, credentials=credentials)
```
Comment (gaya3-zipstack) [inline backend/utils/file_storage_helper.py:20]: If provider is not present, we can default it to Local

File: frontend/src/components/settings/users/Users.jsx
Comment (gaya3-zipstack) [inline frontend/src/components/settings/users/Users.jsx:1]: Are these changes intentional?

File: backend/file_management/file_management_helper.py [hunk 2]
```diff
@@ -141,15 +142,15 @@ def upload_file(
             remote_file.write(file.read())
 
     @staticmethod
-    def fetch_file_contents(file_system: UnstractFileSystem, file_path: str) -> Any:
-        fs = file_system.get_fsspec_fs()
-        try:
-            file_info = fs.info(file_path)
-        except FileNotFoundError:
+    def fetch_file_contents(file_path: str) -> Any:
+        file_storage = FileStorageUtil.initialize_file_storage(
```
Comment (gaya3-zipstack) [inline backend/file_management/file_management_helper.py:146]: I think I need to understand this better.

**Comment by harini-venkataraman** [comment]
Closing, since the changes went in as a part of a different PR
